data/argentium-snek.json

```diff
@@ -1,8 +1,18 @@
 {
-	"name": "Argentium Snek",
+	"names": [
+		"Argentium Snek"
+	],
```

Comment:
It's unnecessary to replace a singular `name` value with a `names` array.

In the future, `names` could be renamed to `alternateNames` to clarify that `name` represents the primary name. However, for now, using `names` with the primary name as the first element in the array is fine.